client/rm.go

```diff
@@ -21,13 +24,21 @@ func (cli *HyperClient) HyperCmdRm(args ...string) error {
 	if len(args) == 0 {
 		return fmt.Errorf("\"rm\" requires a minimum of 1 argument, please provide POD ID.\n")
 	}
-	pods := args
-	for _, id := range pods {
-		err := cli.client.RmPod(id)
-		if err == nil {
-			fmt.Fprintf(cli.out, "Pod(%s) is successful to be deleted!\n", id)
+	for _, id := range args {
+		if opts.Container {
+			err := cli.client.RemoveContainer(id)
+			if err == nil {
+				fmt.Fprintf(cli.out, "container %s is successfully deleted!\n", id)
+			} else {
+				fmt.Fprintf(cli.err, "%v\n", err)
```

Comment:
Will update this. Actually this is copied from the original one, I don't know if you are depends on this behavior.